### ISSUE TITLE
torch squeeze in htl_players causes a crash when testing with only 1 enviornment

### ISSUE THREAD

**Issue by Robokan** [nv-tlabs/ASE#50]
current_action = torch.squeeze(current_action.detach())
around line 91 in hrl_players will cause a crash if you are testing with only on environment. changing it to:
current_action = current_action.detach()

fixes the crash.

**Comment by xbpeng** [comment]
thanks a lot for pointing that out, fixed:
https://github.com/nv-tlabs/ASE/commit/f0e41937095014ec972c7f3ae9f3f15a99978a99

**Comment by Robokan** [comment]
I just found another case in ase_players. Line 88. This one will crash it when testing the llc networks just by itself.
current_action = torch.squeeze(current_action.detach())
should be:
current_action = current_action.detach()

**Comment by xbpeng** [comment]
thanks, fixed
